app/models/sample.rb

```diff
@@ -717,6 +718,11 @@ def kickoff_pipeline
     # only kickoff pipeline when no active pipeline_run running
     return unless pipeline_runs.in_progress.empty?
 
+    if do_not_process
```

Comment:
rails tip: you can write...

`if do_not_process?`

rails will add the `?` method